lib/ui/viewer/file/fading_app_bar.dart

```diff
@@ -272,16 +271,12 @@ class FadingAppBarState extends State<FadingAppBar> {
   Future<void> _handleUnHideRequest(BuildContext context) async {
     final s = SelectedFiles();
     s.files.add(widget.file);
-    Navigator.push(
+    Navigator.pop(context);
```

Comment:
what and why are we popping here.. may be add a comment too?

Reply:
Removing this, thought what this is doing was needed, but it don't make much of a difference so removing.